.github/workflows/build-publish-pypi.yml

```
@@ -61,6 +64,13 @@ jobs:
       - name: Extract Version from Tag
         run: echo "VERSION=${GITHUB_REF#refs/tags/test-v}" >> $GITHUB_ENV
 
+      - name: Set version variable in module file
+        run: echo "VERSION=${{ env.VERSION }}" > src/sindri/__init__.py
```

The dunder name should be `__version__`, the version should be a string, and I don't think we want to be clobbering the contents of `__init__.py` because this will override default exports. Same comment above. Something like this is closer:

```
        run: echo '__version__ = "${{ env.VERSION }}"' >> src/sindri/__init__.py
```

but I think it's cleaner to modify the placeholder version directly, so my actual suggestion is this and the other suggestion below:

```suggestion
        run: sed -i 's/0.0.0/${{ env.VERSION }}/g' src/sindri/__init__.py
```